packages/contracts/src/contracts/current/protocol/Exchange/MixinSettlement.sol

```diff
@@ -16,23 +16,22 @@
 
 */
 
-pragma solidity ^0.4.21;
-pragma experimental ABIEncoderV2;
+pragma solidity ^0.4.23;
 
 import "./mixins/MSettlement.sol";
 import "./mixins/MAssetProxyDispatcher.sol";
-import "./LibOrder.sol";
-import "./LibMath.sol";
-import "../AssetProxy/IAssetProxy.sol";
+import "./lib/LibOrder.sol";
+import "./lib/LibMath.sol";
 
-/// @dev Provides MixinSettlement
 contract MixinSettlement is
     LibMath,
     MSettlement,
     MAssetProxyDispatcher
 {
+    /// ZRX metadata used for fee transfers.
```

Comment:
Can we perhaps extend this comment with why we hard-code this? Just so future dev's know.